vendor/bat-native-ads/src/bat/ads/internal/frequency_capping/exclusion_rules/conversion_frequency_cap.cc

```diff
@@ -0,0 +1,53 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "bat/ads/internal/frequency_capping/exclusion_rules/conversion_frequency_cap.h"
+
+#include "bat/ads/creative_ad_info.h"
+#include "bat/ads/internal/client.h"
+#include "bat/ads/internal/frequency_capping/frequency_capping.h"
+
+#include "base/logging.h"
+#include "base/strings/stringprintf.h"
+
+namespace ads {
+
+ConversionFrequencyCap::ConversionFrequencyCap(
+    const FrequencyCapping* const frequency_capping)
```

Comment:
double const?

Reply:
const X* const p means “p is a const pointer to an X that is const”: you can’t change the pointer p itself, nor can you change the X object via p.

Reply:
@NejcZdovc see https://isocpp.org/wiki/faq/const-correctness#const-ptr-vs-ptr-const